frontend/js/src/common/listens/MBIDMappingModal.tsx

```diff
@@ -285,21 +267,74 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
                 </div>
               )}
             </div>
-            <div className="modal-footer">
-              <button
-                type="button"
-                className="btn btn-default"
-                onClick={closeModal}
-              >
-                Cancel
-              </button>
-              <button
-                type="submit"
-                className="btn btn-success"
-                disabled={!selectedRecording}
+            <div className="modal-footer" style={{ textAlign: "left" }}>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                }}
               >
-                Add mapping
-              </button>
+                <div style={{ textAlign: "left", marginBottom: "10px" }}>
```

Comment:
The style here are not required, I'll comment above about adding a class to the parent element for the bottom margin
```suggestion
                <div>
```